<launch>

    <node pkg = "l1_estimator" type = "estimator_test" name = "estimator_test" output="screen"/>
        <param name="simulation_param_dir" value="$(find l1_estimator)/config/simulation_model.yaml"/>
        <param name="nominal_param_dir" value="$(find l1_estimator)/config/nominal_model.yaml"/>
        
        <param name="quad_model" type="int" value="1"/>
        <param name="simulation_final_time" type="double" value="10"/>
        <param name="discrete_time" type="double" value="0.010"/>
        
        <param name="kp" type="double" value="0.0"/>
        <param name="kv" type="double" value="0.0"/>
        <param name="kq" type="double" value="5.0"/>
        <param name="kw" type="double" value="2.0"/>

        <param name="b_sigma" type="double" value="5"/>
        <param name="e_sigma" type="double" value="10"/>
        
        <param name="b_theta" type="double" value="5"/>
        <param name="e_theta" type="double" value="10"/>

        <param name="Gamma_sigma" type="double" value="10000"/>
        <param name="Gamma_theta" type="double" value="1000"/>
        
        <param name="tau_sigma" type="double" value="10"/>
        <param name="tau_theta" type="double" value="2"/>
        

</launch>